frontends/infinite-corridor/src/api/fields/factories.ts

```diff
@@ -1,7 +1,8 @@
 import { faker } from "@faker-js/faker"
-import * as R from "ramda"
-import { makePaginatedFactory, Factory } from "ol-util"
-import { LearningResourceType, factories } from "ol-search-ui"
+import { makePaginatedFactory, Factory } from "ol-util/build/factories"
```

Comment:
I would very much have preferred an import from `ol-util/factories` instead of `ol-util/build/factories`. It's fairly easy to configure that with Node [subpath exports](https://nodejs.org/api/packages.html#subpath-exports).

But as far as I can tell, making that work with Typescript means moving to ESM-style module resolution, which is a fairly big change. So I went for the uglier `ol-util/build/factories` imports. From https://github.com/microsoft/TypeScript/issues/33079#issuecomment-1216866007 (emphasis mine):

> TypeScript doesn’t yet have a resolution mode suitable for most bundlers, and **package.json exports are not yet configurable independently of the module resolution mode on the whole**. This is discussed at length in https://github.com/microsoft/TypeScript/issues/50152.

Related: https://www.typescriptlang.org/docs/handbook/esm-node.html